### DISCUSSION TITLE
does ColossalAI support tensorflow?

### DISCUSSION THREAD

**Discussion by robin-ai-ml** [hpcaitech/ColossalAI#3510] [Community | Q&A]
I have a tensorflow keras project, now I want to integrate tensorflow project to colossalai framework.
but i read the document
Requirements:

PyTorch >= 1.11 (PyTorch 2.x in progress)
Python >= 3.7
CUDA >= 11.0
so I wonder if Colossal AI framework support tensorflow project?
or it is only run with Pytorch?

**Comment by jackielyc** [comment]
As far as I know, only run with Pytorch now.

**Comment by andrew600-sudo** [comment]
https://trendified.net/
